Sources/StreamChatUI/ChatChannelList/ChatChannelListItemView+SwiftUI_Tests.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  ChatChannelListItemView+SwiftUI_Tests.swift
+//  StreamChatUITests
+//
+//  Created by Igor Rosocha on 3/5/21.
+//  Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
```

Comment:
The formatter didn't work. Please rebase, run `bootstrap.sh` and amend the commit to make the formatter work.